src/RestQuery.js

```diff
@@ -866,7 +866,7 @@ function includePath(config, auth, response, path, restOptions = {}) {
           return set;
         }
       }
-      if (i == (keyPath.length - 1)) {
```

Comment:
Please revert

Reply:
Bruh, i think its prettier pre-commit hook that keep re-adding it !

Reply:
You can just disable that for now, we'll make a separate lint/prettier PR.